src/Resque/Socket/Server.php

```diff
@@ -187,12 +203,7 @@ public function run()
         }
 
         if (function_exists('pcntl_signal')) {
-            // PHP 7.1 allows async signals
-            if (function_exists('pcntl_async_signals')) {
-                pcntl_async_signals(true);
-            } else {
-                declare(ticks = 1);
-            }
+            declare(ticks = 1);
```

Comment:
Here the async feature (merged in #4ca7d32b) got lost. Could you please rebase on the current master to resolve this issue?

Reply:
done